### DISCUSSION TITLE
为什么不把服务端的功能整合进客户端，开发一个即使抓取即使阅读的APP呢？

### DISCUSSION THREAD

**Discussion by CCCAUCHY** [DIYgod/RSSHub#11924] [Ideas]
看起来像是一个浏览器，但是这个特殊的浏览器按照RSS的方式过滤掉了大部分无用的信息，只显示精华的部分。
虽然这么做就无法为其他RSS客户端提供RSS源，但是如果人手一个RSS源呢？哪怕不是部署在服务器上呢？还需要一个中心化的RSS源吗？
不如就叫它Browser power by RSS吧！
issue太多了，中英混杂，没有找到相关的点子，如有重复，请见谅。

**Comment by CCCAUCHY** [comment]
显然，使用特殊修改后的安卓系统的Linux功能部署RSSHub后，就可以实现差不多的效果，但是为什么不一步到位呢？

**Reply by 1638410** [reply]
手机可以直接用termux安装这个（不用docker就好了），不用所谓特殊修改

**Comment by storytellerF** [comment]
即便是最后输出是一个可以直接安装的apk，也可以在实现层面做成网络请求和ui显示分离的结构，本质和rsshub+rss客户端一样。
